openshift/README.md

```diff
@@ -118,6 +125,20 @@ runAsUser:
   type: RunAsAny
 ```
 
+### Log collection
+
+The Datadog Agent's log collection is set up in OpenShift largely the same as other Kubernetes clusters. The Datadog Operator and Helm Chart will mount in the `/var/log/pods` directory that the Datadog Agent pod will use to monitor the logs of the pods and containers on its respective host. However, with the Datadog Operator you do need to apply additional SELinux options in order to give the Agent permissions to read these log files.
+
+See [Kubernetes Log Collection][7] for further general information and the [Distributions][15] page for configuration examples.
+
+### APM
+
+In Kubernetes there are three main options to route the data from the application pod to the Datadog Agent pod. The Unix Domain Socket (UDS), the HostIP:HostPort option (TCP/IP), and the Kubernetes Service. The Datadog Operator and Helm Chart default to the UDS option as this is the most resource efficient, however, this option doesn't work well in OpenShift. As it requires elevated SCC and SELinux options in both the Agent pod and application pod.
+
+Datadog recommends to disable the UDS option explicitly to avoid this, and avoid the Admission Controller injecting this configuration.
```

Comment:
```suggestion
Datadog recommends disabling the UDS option explicitly to avoid this, and to avoid the Admission Controller injecting this configuration.
```